test/screenshot/chips/index.js

```diff
@@ -24,9 +25,9 @@ class ShirtSizes extends React.Component {
   render() {
     return (
       <ChipSet>
-        <Chip selected={this.isSelected(0)} id={0} label='Small' handleSelect={this.handleSelect}/>
-        <Chip selected={this.isSelected(1)} id={1} label='Medium' handleSelect={this.handleSelect}/>
-        <Chip selected={this.isSelected(2)} id={2} label='Large' handleSelect={this.handleSelect}/>
+        <Chip selected={this.isSelected(0)} label='Small' handleSelect={() => this.handleSelect(0)}/>
```

Comment:
I'm seeing a warning when you delete a chip. I think its the ripple is calling setState on unmount. I think this was an issue.

<img width="1193" alt="screen shot 2018-07-24 at 16 45 41" src="https://user-images.githubusercontent.com/579873/43171962-a4759bc4-8f61-11e8-9e8c-309c91d99f2a.png">